ipfix-flow-exporter/bpf_ipfix_egress_user.c

```diff
@@ -132,37 +108,14 @@ int main(int argc, char **argv)
         return EXIT_FAILURE;
     }
 
-    if(attach_tc_egress_filter) {
-	if(chain == true){
-            if (tc_chain_bpf(bpf_map_file_path, filename, egress) != 1) {
-                fprintf(stderr, "ERR: tc chain egress failed \n");
-                close_logfile();
-                 return EXIT_FAILURE;
-            }
-	}
-        else {
-            if (tc_attach_filter(if_name, filename, EGRESS, egress) != 1) {
-                fprintf(stderr, "ERR: TC EGRESS filter attach failed\n");
-                close_logfile();
-                return EXIT_FAILURE;
-            }
-       }
-       if (populate_egress_fds() == EXIT_FAILURE) {
-            fprintf(stderr, "ERR: Fetching TC EGRESS maps failed\n");
-            close_logfile();
-            return EXIT_FAILURE;
-       }
-       if (remote_ip == NULL) {
-                log_info("Remote IP is not configured by user, so configuring localhost as remote_ip");
-                remote_ip = "127.0.0.1" ;
-       }
+    if (populate_egress_fds() == EXIT_FAILURE) {
+        fprintf(stderr, "ERR: Fetching TC EGRESS maps failed\n");
+        close_logfile();
+        return EXIT_FAILURE;
     }
-
-    if (remove_egress_tc_filter) {
-        log_debug("TC remove egress filters on device %s",
-                   if_name);
-        tc_remove_filter(if_name, EGRESS);
-        return EXIT_SUCCESS;
+    if (remote_ip == NULL) {
+        log_info("Remote IP is not configured by user, so configuring ipv4 localhost as remote_ip");
```

Comment:
```suggestion
        log_info("Remote IP is not configured by user, so configuring IPv4 loopback address as remote_ip");
```

Reply:
Personally I think it's bad form to default to IPv4, and it'd be better to fail than default to IPv4.